src/main/scala/org/broadinstitute/dsde/workbench/sam/google/GoogleKeyCache.scala

```diff
@@ -39,39 +39,6 @@ class GoogleKeyCache(
   val utf8Charset = Charset.forName("UTF-8")
 
   override def onBoot()(implicit system: ActorSystem): IO[Unit] = {
-    val projectTopicName = ProjectTopicName
-      .newBuilder()
-      .setProject(googleServicesConfig.googleKeyCacheConfig.monitorPubSubConfig.project)
-      .setTopic(googleServicesConfig.googleKeyCacheConfig.monitorPubSubConfig.topic)
-      .build()
-    new GooglePubSubMonitor(
-      googleKeyCachePubSubDao,
-      googleServicesConfig.googleKeyCacheConfig.monitorPubSubConfig,
-      googleServicesConfig.serviceAccountCredentialJson,
-      new GoogleKeyCacheMessageReceiver(googleIamDAO)
-    ) {
-      override protected def init: IO[Unit] = for {
-        _ <- super.init
-        _ <- IO.fromFuture(
-          IO(
-            googleKeyCachePubSubDao.setTopicIamPermissions(
-              projectTopicName.getTopic,
-              Map(googleServicesConfig.projectServiceAccount -> "roles/pubsub.publisher")
-            )
-          )
-        )
-        _ <- IO.fromFuture(
-          IO(
-            googleStorageDAO.setObjectChangePubSubTrigger(
-              googleServicesConfig.googleKeyCacheConfig.bucketName,
-              projectTopicName.toString,
-              List("OBJECT_DELETE")
-            )
-          )
-        )
-      } yield ()
-    }.startAndRegisterTermination()
```

Comment:
This moved because I realized it changed in the last PR from eager code that started an actor to an IO which if left here is just dropped on the floor. This means monitor never actually starts!